packages/ui/src/utils/setUserAgent/setUserAgent.ts

```diff
@@ -60,18 +68,36 @@ export const setUserAgent = ({
   ];
 
   switch (componentName) {
-    case 'AccountSettings': {
-      // remove cast when Category input types are available
-      input = ACCOUNT_SETTINGS_INPUT_BASE as SetCustomUserAgentInput;
+    case 'Authenticator': {
+      input = { ...AUTHENTICATOR_INPUT_BASE, additionalDetails };
+      break;
+    }
+    case 'ChangePassword': {
+      input = { ...ACCOUNT_SETTINGS_INPUT_BASE, additionalDetails };
+      break;
+    }
+    case 'DeleteUser': {
+      input = { ...ACCOUNT_SETTINGS_INPUT_BASE, additionalDetails };
       break;
     }
```

Comment:
In this case though I think the `additionalDetails` should still show either `DeleteUser` or `ChangePassword` because its type requires you use the component name

Reply:
Needs to be the parent `AccountSettings` per JS request